registration/templates/email/confirm_verification_body.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+{% blocktranslate %}
+Hello {{ first_name }},
+{% endblocktranslate %}
+
+{% blocktranslate %}
+We have received your application and verified your data. Your account is now fully activated.
+{% endblocktranslate %}
+
+{% blocktranslate %}
+You can now login and file a license request.
+{% endblocktranslate %}
+
+
+{% block login_link %}
+https://{{ domain }}{% url 'home' %}
```

Comment:
When I tried to open the link, it currently fails. I think this is caused due to a missing certificate for https.

Reply:
I want to keep it as secure as possible and only present https here. I have dynamically created the domain though.